src/box/lua/load_cfg.lua

```diff
@@ -219,40 +219,15 @@ local template_cfg = {
     txn_timeout           = 'number',
 }
 
-local function normalize_uri(port)
-    if port == nil or type(port) == 'table' then
-        return port
-    end
-    return tostring(port);
-end
-
 local function normalize_uri_list_for_replication(port_list)
-    local result = {}
     if type(port_list) == 'table' then
-        for _, port in ipairs(port_list) do
-            table.insert(result, normalize_uri(port))
-        end
-    else
-        table.insert(result, normalize_uri(port_list))
-    end
-    return result
-end
-
-local function normalize_uri_list_for_listen(port_list)
-    local result = {}
-    if type(port_list) == 'table' then
-        for _, port in ipairs(port_list) do
-            table.insert(result, normalize_uri(port))
-        end
-    else
-        return normalize_uri(port_list)
+        return port_list
     end
-    return result
+    return {port_list}
 end
```

Comment:
I don't see where this feature about passing parameters to `parse/parse_many` is used. Why is it needed?

Reply:
The idea was to extend URI's library so that it could parse the same as box.cfg.listen and box.cfg.replication.